sdk/storage/perf-tests/storage-blob-track-1/README.md

```diff
@@ -0,0 +1,14 @@
+### Guide
+
+1. Navigate to `sdk\storage\perf-tests\storage-blob-track-1`
+2. Do `rush update`.
```

Comment:
should one rebuild storage-blob itself as well? e.g. `rush rebuild -t @azure/storage-blob` ?

Reply:
This is track-1(Version 10), we don't maintain it in our repo.
So, building `storage-blob` is not needed for track-1.

`npm run setup` in the next line, builds the `test-utils-perfstress` project, and installs it here.

(Also, track-1 perf tests are not maintained by rush, they are independent npm projects.)